### ISSUE TITLE
Question about checkpoint

### ISSUE THREAD

**Issue by Jhonve** [art-programmer/FloorNet#10]
Did you pre-train the network and supply the pre-trained model in the floder named 'checkpoint'?

I tried to use your checkpoint to test your data, but I got very bad prediction result which is as same as the input point cloud.(python train.py --task=test --startIteration=10000)

I tried to train by myself too. However, I found that there are lots of data which is not match between point cloud and labels in the training data. (I got your tfrecord data from Mega, “Tango_train.tfrecords”)Did I train the network in a wrong way?
![data_not_match_1](https://user-images.githubusercontent.com/26267554/47367689-c3d99580-d712-11e8-8453-533ffacd8fc1.png)
![[data_not_match_2](https://user-images.githubusercontent.com/26267554/47367712-cd62fd80-d712-11e8-96b8-540b91628ae8.png)

**Comment by art-programmer** [comment]
It's strange. Could you please double-check if you load the pre-trained model successfully? The visualization also seems weird, the point cloud (white dots) is mis-aligned. Could you please check if the downloaded file is complete?
